### PR TITLE
Science page

### PR DESCRIPTION
**Pull request by vivianha534**
**Updates to existing files**
- updated SciencePage.js to include info cards and a second header
- updated margins of the SciencePage to be smaller so that the content takes up more space
- created media query in SciencePage.css so that the vertical nav only appears on desktop and not mobile devices

**Info Card**
- InfoCard is set up assuming that the parent page has an array of objects that it will map into InfoCard. The object should have key value pairs for title, description, imgUrl, and imgAlt. (see SciencePage.js for more info)
- InfoCard currently sets the id attribute using regex to delete all spaces from the article title and lowercase the article title (ie. title: "Article 1" -> id="article1")

**Vertical Nav**
- Doesn't show up for screens smaller than 1024px (desktop)
- Hard coded anchor tag to jump to top of page so it's important that the div at the top of the page has an id="page-top"
- Takes an array of objects and uses regex to set up the anchor tags (same idea as setting up the id attribute for the InfoCard)
- Button turns blue, and text appears to the side on hover
- Only show after you've scrolled 700px

### CODE REVIEW

File: src/components/VerticalNav.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <span id="hover-text">
+          Top of page
+        </span>
+      </div>
+      
+      {infoCards.map((cardObj) => 
+        <div id="hover-div">
+          <a href={`#${cardObj.title.replace(/\s+/g, '').trim().toLowerCase()}`}>
+            <button className="nav-button"></button>
+          </a>
```
Comment (darren2hang) [inline src/components/VerticalNav.js:22]: I don't think we really need a button here. Like maybe give the a tag a class and apply the nav-button properties to that class.

File: src/styles/InfoCard.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.info-card-reverse{
+    flex-direction: row-reverse;
+}
+
+/* This is to account for the fixed header when jumping to the articles
+without this styling the fixed header would block the title */
+.hash{
+    height:50px;
```
Comment (darren2hang) [inline src/styles/InfoCard.css:16]:
```suggestion
    height: var(--navbar-height);
```

Comment (darren2hang) [reply src/styles/InfoCard.css:16]: I made a variable for the navbar height.

**Review by darren2hang — CHANGES_REQUESTED** [review]
Nice work! I think you did a good job of separating the logic into multiple components. I left a few nits.
I think the styling of the vertical navbar can be improved a bit, like remove the border and outline and also making it more narrow, but that's just a design preference and can be addressed later.

**Review by darren2hang — APPROVED** [review]

**Comment by vivianha534** [comment]
This has already been merged into setUp

**Pull request closed without merging**